test/extension/run_at_cartodb_schema.sql

```diff
@@ -1,7 +0,0 @@
-SET SCHEMA 'cartodb';
```

Comment:
why is this file deleted? it is referenced from the tests:
```
Makefile:       bash test/extension/test.sh
```

Reply:
oh, I see they were moved to `load_sql_file_schema` calls

Reply:
Yes, since `PARALLEL` needs to be stripped for 9.5- I couldn't just load the file directly.